.claude/skills/sync-claude-docs/SKILL.md

```
@@ -0,0 +1,177 @@
+✅ 모든 문서가 코드베이스와 일치합니다.
+```
+
+---
+
+## Step 4. 수정 계획 확인
+
+실제 파일 수정 전에 변경 계획을 출력합니다.
+
+```
+✏️ 수정 계획
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)